enterprise/cmd/executor/internal/config/config.go

```diff
@@ -179,8 +181,18 @@ func getKubeConfigPath() string {
 }
 
 func (c *Config) Validate() error {
-	if c.QueueName != "" && c.QueueName != "batches" && c.QueueName != "codeintel" {
+	if c.QueueName == "" && c.QueueNames == "" {
+		c.AddError(errors.New("One of EXECUTOR_QUEUE_NAME and EXECUTOR_QUEUE_NAMES should be set"))
```

Comment:
```suggestion
		c.AddError(errors.New("Neither EXECUTOR_QUEUE_NAME or EXECUTOR_QUEUE_NAMES is set"))
```

Slight change to verbage. `One of` didn't roll off the tongue very well for me